sdk/servicebus/service-bus/test/serviceBusClient.spec.ts

```diff
@@ -311,79 +301,101 @@ describe("Errors with non existing Queue/Topic/Subscription", async function():
   });
 });
 
-// describe("Test createFromAadTokenCredentials", function(): void {
-//   let sbClient: ServiceBusClient;
-//   let errorWasThrown: boolean = false;
-
-//   const env = getEnvVars();
-//   const serviceBusEndpoint = (env.SERVICEBUS_CONNECTION_STRING.match(
-//     "Endpoint=sb://((.*).servicebus.windows.net)"
-//   ) || "")[1];
-
-//   async function testCreateFromAadTokenCredentials(host: string, tokenCreds: any): Promise<void> {
-//     const testMessages = TestMessage.getSample();
-//     sbClient = ServiceBusClient.createFromAadTokenCredentials(host, tokenCreds);
-//     sbClient.should.be.an.instanceof(ServiceBusClient);
-//     const clients = await getSenderReceiverClients(
-//       sbClient,
-//       TestClientType.UnpartitionedQueue,
-//       TestClientType.UnpartitionedQueue
-//     );
-
-//     const sender = clients.senderClient.createSender();
-//     const receiver = await clients.receiverClient.createReceiver(ReceiveMode.peekLock);
-//     await sender.send(testMessages);
-//     const msgs = await receiver.receiveMessages(1);
-
-//     should.equal(Array.isArray(msgs), true, "`ReceivedMessages` is not an array");
-//     should.equal(msgs[0].body, testMessages.body, "MessageBody is different than expected");
-//     should.equal(msgs.length, 1, "Unexpected number of messages");
-//   }
-
-//   it("throws error when using `CreateFromAadTokenCredentials` in browser #RunInBrowser", async function(): Promise<
-//     void
-//   > {
-//     // We use the `!isNode` check here to ensure this test is run only in browser only
-//     // as by default all tests run in Node
-//     if (!isNode) {
-//       const credentials: any = {};
-//       await testCreateFromAadTokenCredentials(serviceBusEndpoint, credentials).catch((err) => {
-//         errorWasThrown = true;
-//         should.equal(
-//           err.message,
-//           "`createFromAadTokenCredentials` cannot be used to create ServiceBusClient as AAD support is not present in browser."
-//         );
-//       });
-//       should.equal(errorWasThrown, true, "Error thrown flag must be true");
-//     }
-//   });
-
-//   it("throws error for invalid tokenCredentials", async function(): Promise<void> {
-//     await testCreateFromAadTokenCredentials(serviceBusEndpoint, "").catch((err) => {
-//       errorWasThrown = true;
-//       should.equal(
-//         err.message,
-//         "'credentials' is a required parameter and must be an instance of ApplicationTokenCredentials | UserTokenCredentials | DeviceTokenCredentials | MSITokenCredentials.",
-//         "ErrorMessage is different than expected"
-//       );
-//     });
-//     should.equal(errorWasThrown, true, "Error thrown flag must be true");
-//   });
-
-//   it("Coerces input to string for host in createFromAadTokenCredentials", async function(): Promise<
-//     void
-//   > {
-//     const tokenCreds = await getTokenCredentialsFromAAD();
```

Comment:
Now that we don't use `getTokenCredentialsFromAAD()` anymore in this file, please remove the commented imports for the same.